### ISSUE TITLE
Use the wall mounted tablet for HA as well as this integration?

### ISSUE THREAD

**Issue by Backslashh** [Bre77/myair#17]
First of all, thank you for this component.
I know this is now part of core but just wanted to find out something.

We are building our first house soon and it will be coming with the myAir system. I was planning on using the wall mounted tablet as a HA tablet. But if I am using the tablet to display HA (with fullyKiosk or something) does this integration still work?

Is it just that the myPlace app has to be in the background or does it need to be in the foreground for the integration to work?

Bottom line is - can this integration still work if the tablet is displaying HA or of its being used for something else (YouTube, etc)?

**Comment by Bre77** [comment]
Yes, I run home Assistant on the wall mounted tablet and the service runs in the background just fine. They have gone to some lengths to ensure the service is always running and never stopped.

**Comment by Backslashh** [comment]
Ah that's brilliant. 
Thanks for getting back to me. Truly appreciate it.
